cmd/dm-ctl/main.go

```diff
@@ -63,6 +66,40 @@ func main() {
 		os.Exit(2)
 	}
 
+	if len(cmdArgs) > 0 {
+		commandMode(cmdArgs)
+	} else {
+		interactionMode()
+	}
+}
+
+func collectArgs(args []string) []string {
+	collectedArgs := make([]string, 0, len(args))
+	for i := 0; i < len(args); i++ {
+		switch strings.ToLower(args[i]) {
+		case "start-task", "query-status", "stop-task", "pause-task", "resume-task", "show-ddl-locks", "unlock-ddl-lock", "break-ddl-lock":
```

Comment:
Is there any method to obtain these (all available) command list in _cobra_?

and can we show these commands in `./dmctl --help`?

Reply:
let me try